Core/DotLanches.Domain/Entities/Pedido.cs

```diff
@@ -1,6 +1,14 @@
+using System.Text.Json.Serialization;
+
 namespace DotLanches.Domain.Entities;
 
 public class Pedido
 {
-    
+    public int Id { get; set; }
+    public DateTime CreatedAt { get; set; }
+    public string? ClienteCPF { get; set; }
+    public decimal TotalPrice { get; set; }
```

Comment:
Essa propriedade poderia ser calculada automaticamente, não?
```suggestion
        public decimal TotalPrice => Combos.Sum(x => x.Price);
```